features/Steps/LabelSteps.php

```diff
@@ -40,48 +40,66 @@ public function iCreateAnInvisibleLabelWithARandomNameOfCharacters(int $nrOfChar
     public function labelsTestDataIsAvailable(): void
     {
         // Create "public-visible" if it doesn't exist yet and (re)set the right privacy and visibility in case its needed
-        $this->createLabel('public-visible', true, true);
         $this->getLabel('public-visible');
+        if ($this->responseState->getStatusCode() === 404) {
+            $this->createLabel('public-visible', true, true);
```

Comment:
If the label needs to be created there is no need to patch the privacy and visibility.

Reply:
Nice find.